airflow/www/templates/airflow/dags.html

```diff
@@ -215,13 +215,13 @@ <h2>{{ page_title }}</h2>
                   title="Status of all previous DAG runs.">info</span>
           </th>
           <th>Schedule</th>
-          <th style="width:180px;">{{ sortable_column("Last Run", "last_dagrun") }}
+          <th style="width:180px;">{{ sortable_column("Last Data Interval", "last_dagrun") }}
             <span class="material-icons text-muted js-tooltip" aria-hidden="true"
-                  title="Date/Time of the latest Dag Run.">info</span>
+                  title="Starting Date/Time interval of the latest Dag Run.">info</span>
```

Comment:
I find this confusing. I'd prefer we don't change the title (leave it "Last Run") and change this to "Logical Date of the latest Dag Run.".

Reply:
Yeah makes sense to me because it is theoretically the logical date :)